README.md

```diff
@@ -268,6 +268,36 @@ requires an above average amount of memory available. To avoid JS Heap overflows
 npm scripts that run webpack (`start`, `start:dev`, and `start:poll`) pass an argument
 `--max_old_space_size` to increase the memory allocation. This can be adjusted as necessary.
 
+## Release Checklist
+The following conditions must be tested before deploying a release. If any of the following
+circumstances causes a break, CHANGELOG should indicate a workaround or migration steps
+should be provided.
+
+- [ ] Pull into an existing environment
+  - [ ] Try a simple restart of the server (`npm run start`)
+  - [ ] Try with a fresh `node_modules`
+  - [ ] Try with a fresh configs (usually `~/.bpanel`) directory
+  - [ ] Install and uninstall new remote plugins using bpanel-cli
+  - [ ] Install and uninstall new local plugins using bpanel-cli
+- [ ] Download zip of development branch from GitHub (this emulates downloading a release from the website).
+Test the following in a fresh environment (i.e. with no existing ~/.bpanel or other configs)
+  - [ ] Run `npm install` and confirm: it creates a new ~/.bpanel directory with `configs/`, `configs.js`,
+  and `local_plugins`
+  - [ ] Run `docker-compose up` and confirm after build is complete that bPanel is accessible at `localhost:5000`
+  - [ ] Confirm fresh install comes with the correct default plugins
```

Comment:
Do you want to list the defaults? Or is it in flux?

Reply:
I went back and forth on this. It has the potential to change. I thought of maybe listing them and putting an "AS OF: [DATE]", but that doesn't even really tell you much. So I figured leave it ambiguous and assume that the maintainer actually knows what this is, and in fact this kind of forces you to check.